knowledge-base/grid-line-breaks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Render Line Breaks in Grid Column
+description: How to render new lines inside Grid cell.
+type: how-to
+page_title: Render Line Breaks in Grid Column
+slug: grid-kb-line-breaks
+position: 
+tags: 
+ticketid: 1650530
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+This knowledge base article answers the following questions:
+
+* How do I display line breaks inside a Grid column?
+* Is it possible to render HTML inside a Grid column?
+
+## Solution
+
+1. To create a new line break in the HTML content of a Grid cell, replace the `\n` segment with the `<br>` HTML tag. 
+2. To render the `<br>` HTML tag from a string, use a [`MarkupString`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.markupstring?view=aspnetcore-8.0). 
```

Comment:
Our style guide's latest recommendation is to go with this syntax for external links. [Source.](https://docs.telerik.com/style-guide/cross-references#guidelines)
```suggestion
2. To render the `<br>` HTML tag from a string, use a <a href="https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.markupstring?view=aspnetcore-8.0" target="_blank"><code>MarkupString<code></a>. 
```